src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/MvcCoreMvcOptionsSetup.cs

```diff
@@ -83,6 +84,11 @@ public void Configure(MvcOptions options)
             options.ModelValidatorProviders.Add(new DefaultModelValidatorProvider());
         }
 
+        public void PostConfigure(string name, MvcOptions options)
```

Comment:
What's the reason for post configure? Is the idea to make this the last mmdp?

Reply:
Should document this with some comments YO. You don't want someone to come through and *simplify* this later.